recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 build:
   number: 1
   script:
-    - export CMAKE_BUILD_PARALLEL_LEVEL=${CPU_COUNT}
+    - export NPY_NUM_BUILD_JOBS=${CPU_COUNT}
```

Comment:
PS: @HDembinski, if you look in the history, you'll see I forgot this was different in the initial PR ;)